getting-started/windows-calico/demo.md

```diff
@@ -0,0 +1,267 @@
+
+## Create pods on Linux nodes
+
+First, create a client (busybox) and server (nginx) pod on the Linux nodes:
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: calico-demo
+
+---
+
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: busybox
+  name: busybox
+  namespace: calico-demo
+spec:
+  containers:
+  - args:
+    - /bin/sh
+    - -c
+    - sleep 360000
+    image: busybox:1.28
+    imagePullPolicy: Always
+    name: busybox
+  nodeSelector:
+    beta.kubernetes.io/os: linux
+
+---
+
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: nginx
+  name: nginx
+  namespace: calico-demo
+spec:
+  containers:
+  - name: nginx
+    image: nginx:1.8
+    ports:
+    - containerPort: 80
+  nodeSelector:
+    beta.kubernetes.io/os: linux
+EOF
+```
+
+## Create pods on Window nodes
+
+Next, we'll create a client (powershell) and server (porter) pod on the Windows nodes. First the create the powershell pod.
+
+>**Note**: The powershell and porter pod manifests below use images based on `mcr.microsoft.com/windows/servercore:1809`.
+If you are using a more recent Windows Server version, update the manifests to use a [servercore image](https://hub.docker.com/_/microsoft-windows-servercore)
+that matches your Windows Server version.
+{: .alert .alert-info}
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pwsh
+  namespace: calico-demo
+  labels:
+    app: pwsh
+spec:
+  containers:
+  - name: pwsh
+    image: mcr.microsoft.com/windows/servercore:1809
+    args:
+    - powershell.exe
+    - -Command
+    - "Start-Sleep 360000"
+    imagePullPolicy: IfNotPresent
+  nodeSelector:
+    kubernetes.io/os: windows
+EOF
+```
+
+Next, we'll create the porter server pod:
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  name: porter
+  namespace: calico-demo
+  labels:
+    app: porter
+spec:
+  containers:
+  - name: porter
+    image: calico/porter:1809
+    ports:
+    - containerPort: 80
+    env:
+    - name: SERVE_PORT_80
+      value: This is a windows preview demo. Have a good time.
+    imagePullPolicy: IfNotPresent
+  nodeSelector:
+    kubernetes.io/os: windows
+EOF
+```
+
+## Check connectivity between pods on Linux and Windows nodes
+
+Now that client and server pods are running on both Linux and Windows nodes, let's verify that client pods
+on Linux nodes can reach server pods on Windows nodes. First, we will need the porter pod IP:
+
+```bash
+kubectl get po porter -n calico-demo -o 'jsonpath={.status.podIP}'
+```
+
+Then we can exec into the busybox pod and try reaching the porter pod on port 80:
+
+```bash
+kubectl exec -n calico-demo busybox -- nc -vz <porter_ip> 80
+```
+
+To combine both of the above steps:
+
+```bash
+kubectl exec -n calico-demo busybox -- nc -vz $(kubectl get po porter -n calico-demo -o 'jsonpath={.status.podIP}') 80
```

Comment:
Would `wget ip:port -O -` be a better option?

Reply:
I originally had `wget` but I felt like `nc` has better output -- instead of saying the operator was completed, it says the connection was open.

Reply:
I see. Yeah. Happy with `nc`